src/components/05_pages/Roles/Roles.js

```diff
@@ -1,39 +1,18 @@
 import React, { Component } from 'react';
-import makeCancelable from 'makecancelable';
+import { func, arrayOf, object } from 'prop-types';
+import { connect } from 'react-redux';
 import { Link } from 'react-router-dom';
+import LoadingBar from 'react-redux-loading-bar';
+import { requestRoles } from '../../../actions/application';
 
-import Loading from '../../02_atoms/Loading/Loading';
 import { Table, TBody, THead } from '../../01_subatomics/Table/Table';
-import Error from '../../02_atoms/Error/Error';
 
 const Roles = class Roles extends Component {
-  state = {
-    roles: {},
-    loaded: false,
-  };
-  componentDidMount() {
-    this.cancelFetch = makeCancelable(this.fetchData());
+  componentWillMount() {
```

Comment:
I think we should still cancel using https://redux-saga.js.org/docs/api/#canceltask

Reply:
@dawehner I had a look at this stuff. It sort of looks like whatever xhr library you are using needs to support some sort of cancel/abort method, which I don't think `fetch` does.

Reply:
https://stackoverflow.com/questions/46429081/react-redux-saga-event-channel-cancel seems to do that (see first answer)

Reply:
After reading https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html#fetching-external-data I think we should continue using componentDidMount

Reply:
Yes, sorry. This was an oversight on my part.